test/test_image.py

```diff
@@ -273,8 +273,9 @@ def test_write_file_non_ascii():
 ])
 def test_read_1_bit_png(shape):
     with get_tmp_dir() as root:
+        np_rng = np.random.RandomState(0)
```

Comment:
as a very minor nit (nitpick =  feel free not to address): the declaration of `np_rng` doesn't need to be within the context manager. Same below

Reply:
so is the rest of the test :p strictly speaking, only `image_path = os.path.join(root, f'test_{shape}.png')` needs to be there

Reply:
Well we do need `image_path` to be valid, so technically we could only remove the last 2 lines. But yeah, it's a nit anyway :)